.github/workflows/dart_ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Verify formatting
         run: pub run dart_dev format --check
-        if: always() && ${{ matrix.sdk }} == '2.7.2' && steps.install.outcome == 'success'
+        if: always() && matrix.sdk == '2.7.2' && steps.install.outcome == 'success'
```

Comment:
I changed this since this was invalid syntax; `if` entries are interpolated already (which is why we can do `steps.install.outcome` on this same line without that interpolation), and adding interpolation within them messes things up.